internal/builtinfunctions/functions.go

```diff
@@ -240,14 +249,14 @@ func getStringToBoolFunction() schema.CallableFunction {
 			schema.NewStringSchema(
 				nil,
 				nil,
-				regexp.MustCompile(`(?i)^(?:true|false|[tf]|[01])$`),
+				regexp.MustCompile(`(?i)^(?:true|false|[tf01])$`),
 			)},
 		schema.NewBoolSchema(),
 		true,
 		schema.NewDisplayValue(
 			schema.PointerTo("stringToBool"),
 			schema.PointerTo(
-				"Interprets the input as a boolean.\n"+
+				"Returns a boolean by interpreting the input.\n"+
```

Comment:
This is an exception in that your description doesn't mention the *type* of the input value. Again, a consistent pattern is nice... but this also makes me think that, given the context and the name of the function, maybe repeating the value is redundant and we could go with just "input" for all of them?